src/main/java/net/openhft/chronicle/bytes/ref/AbstractReference.java

```diff
@@ -70,7 +70,6 @@ protected void acceptNewBytesStore(@NotNull final BytesStore bytes)
         if (this.bytes != null) {
             this.bytes.release(this);
         }
-        bytes.readInt(offset); // wake it up if it's not pointing to anything yet.
```

Comment:
This actually seems to be called on line 50 anyway (with the correct offset), I think it's redundant here?